common/evidence.py

```diff
@@ -50,3 +50,42 @@ def initialize_sparksession() -> SparkSession:
     )
 
     return spark
+
+
+def read_path(path: str, spark_instance) -> DataFrame:
```

Comment:
Are we sure we need such an elaborate mechanism to to handle paths? Also what this function does is not exactly "reading path" in my opinion.

Reply:
It's a one stop shop function that we use in the metrics repo to be able to read any sort of input.
I thought we could benefit from having such a common function that we could make use of in every parser. Please let me know your concerns.
About the name, would `read_path_to_df` be better?

Reply:
Nah, it's not really important. In a way it makes sense. We can keep using it as it is.